firmware/laseramp.ino

```diff
@@ -1,143 +1,251 @@
+/*
+Release 0.1
+We ignore N commands.
+
+Todo:
+Support for N\d+ line prefix.
+Set on run time the size of bed, the steps per mm
+Save configuration to eeprom
+Save to eeprom default values.
+Finish Raster mode.
+Support for negative values in X, Y, Z.
+Add Homing option for HARDWARE positioning and WORK positioning
+
+http://www.deskcnc.com/Homing.html
+Raster Mode
+G5 <[0-9A-F]>
+each char is a 8bit long code, each command may send up to 64 bytes.cada
```

Comment:
I think I don't understand the intend behind 'raster mode'. Can you explain a bit more how it's supposed to work?
(I was imagining that you'd send some string like G5 5d1f034a2ba76821943d1e7aca603, which would represent either an entire bitmap or a column of a bitmap; in which case, indeed, each two characters actually represents a byte.)